src/model/immutable/EditorState.js

```diff
@@ -440,7 +440,7 @@ class EditorState {
       inlineStyleOverride: null,
       lastChangeType: 'undo',
       nativelyRenderedContent: null,
-      selection: currentContent.getSelectionBefore(),
+      selection: currentContent.getSelectionBefore().set('hasFocus', true),
```

Comment:
Should [EditorState.forceSelection](https://draftjs.org/docs/api-reference-editor-state.html#forceselection) be used here instead?